app/assets/javascripts/controllers/schedule/schedule_form_controller.js

```diff
@@ -260,6 +280,22 @@ ManageIQ.angular.app.controller('scheduleFormController', ['$http', '$scope', 's
     }
   };
 
+  var checkFormDataRetrieval = function(vm) {
+    $q.all(allApiPromises)
+      .then(function() {
+        retrievedFormData(vm);
+      });
+  };
+
+  function retrievedFormData(vm) {
+    vm.afterGet = true;
```

Comment:
`$scope.afterGet` in this controller, but not needed becasue that's only for the initial load.